### PR TITLE
Fix2x2 docs080

### PR DESCRIPTION
**Pull request by rosanneturner**
small changes made in spring to 2x2 docs code, let's merge and clean up our branches

### CODE REVIEW

File: R/safe2x2Test.R [hunk 1]
```diff
@@ -67,15 +35,14 @@ GetExpectedCapitalGrowthSimpleSWithAdjustment <- function(transl, H1set.neutral,
   H1set.transl <-
     H1set.neutral + rbind(rep(transl, 2), rep(-transl, 2))
 
-  # TODO(Rosanne): Zou je dit kunnen controleren en een betere naam kunnen geven?
-  helpFunc <- function(h) {
+  calculate_p_theta_1 <- function(h) {
```
Comment (AlexanderLyNL) [inline R/safe2x2Test.R:38]: Preferably no underscores in names. I think that camelCasing is better.

File: R/safe2x2Test.R [hunk 2]
```diff
@@ -781,19 +746,16 @@ simulateFisherSpreadSampleSizeOptionalStopping <- function(deltaDesign, alpha, p
...
           reject.F[m] <- somePValue <= alpha
         }
 
         percent.reject.F[i] <- mean(reject.F)
       }
 
       if (min(percent.reject.F) >= power) {
-        # TODO(Rosanne): Hier heb ik FALSE van gemaakt klopt dat?
-        not.satisfied <- F
+        not.satisfied <- FALSE
```
Comment (AlexanderLyNL) [inline R/safe2x2Test.R:758]: The dot notation is also not recommended anymore. I use a dot only to mimic `t.test` and `cor.test`, but those are aliases.

**Review by AlexanderLyNL — CHANGES_REQUESTED** [review]
Style comments.

**Comment by rosanneturner** [comment]
@AlexanderLyNL I have made those particular style changes; I propose to wait with adapting the rest of my part to camelcase untill I have made the major changes for the E variables, the code structure is going to change entirely anyway. Shall we now merge this branch, and the other one I requested? Then I'll make a fresh new branch from development to implement all the major changes I made with Peter!

**Pull request merged**